src/tango/aio/fd_aio.h

```diff
@@ -240,7 +240,7 @@ FD_FN_PURE static inline fd_aio_send_func_t fd_aio_send_func( fd_aio_t * aio ) {
    the build system. */
 
 static inline int
-fd_aio_send( fd_aio_t *          aio,
+fd_aio_send( fd_aio_t const *    aio,
              fd_aio_pkt_info_t * batch,
```

Comment:
Actually, fd_aio_pkt_info_t needs const too, but we can do that in the next set of changes